### ISSUE TITLE
Provide a `| string` filter

### ISSUE THREAD

**Issue by janorga** [kestra-io/kestra#5528]
### Feature description

Like it exists a `| number` Pebble filter that converts from a number in string representation to a number (INT). The reverse does not exist: converting an INT variable into a STRING.

This request is about creating the Pebble filter: `| string` which converts from number to string.

**Comment by anna-geller** [comment]
I feel the `| toJson` might already do the trick as it converts any object into a JSON string, but we can still add it, thx for the idea 👍

**Comment by RashmitTopG** [comment]
Can i try this ??

**Comment by AkhilBabuMK** [comment]
i would like to work this issue if it is not already solved. Can i try it ?

**Comment by anna-geller** [comment]
@AkhilBabuMK let's give @RashmitTopG a chance first, he has asked first. Feel free to check [one of these issues](https://github.com/search?q=org%3Akestra-io+label%3A%22good+first+issue%22+is%3Aopen+no%3Aassignee+&type=issues&utm_source=GitHub&utm_medium=github&utm_campaign=hacktoberfest2024&utm_content=Good+First+Issues) 👍

**Comment by hashimzs** [comment]
hello @anna-geller can i work on this issue since the original assignee is not responsive

**Comment by anna-geller** [comment]
Good idea, perhaps you can start from the same branch to keep their commits too? https://github.com/kestra-io/kestra/pull/5585

**Comment by hashimzs** [comment]
@anna-geller  i have created a pr,
 sorry i didnt start from his branch since i worked on it before
